featurize_pdbbind_cluster_job.py

```diff
@@ -86,6 +96,63 @@ def featurize_job(pdb_directories, pickle_out):
   with open(pickle_out, "wb") as f:
     pickle.dump(feature_vectors, f)
 
+def featurize_3d_grid(pdb_directories, pickle_out, box_width, voxel_width, tmp_dir):
+  feature_vectors = {}
+  for count, dir in enumerate(pdb_directories):
```

Comment:
Can you rename dir to something else? In general, it's good to avoid naming local variables that conflict with python primitives (like file, type, dir, etc.)